### ISSUE TITLE
[Acc] Button tooltip doesn't show up on tab\

### ISSUE THREAD

**Issue by setaskin** [microsoft/vscode#132344]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.60.0 (user setup) 
- OS Version: Windows 10 

Steps to Reproduce:

1. Navigate through VS Code Search tab  and tab through all the buttons . ( refresh , clear search results , etc. ) ( Search tab is just an example I can see the same behavior in all buttons) 

Expected : On the tab navigation to each of the button, the tooltip should be displayed. On escape the tooltip should be dismissed. Keyboard users will not know what is the use of the button through the icon for the button.

Actual :  User cannot see the tooltip when they tab into a button. They can only see the tooltip when they hover over. 
![Animation2](https://user-images.githubusercontent.com/37163639/132100379-95cae628-2247-4d5f-a812-20018a9bf9b0.gif)

**Comment by setaskin** [comment]
@isidorn , Hey, I search to see if there is something already open but couldn't find it I hope it is not a duplicate.

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52016
Reasons why we are not changing this still hold which I mentioned [here](https://github.com/Microsoft/vscode/issues/52016#issuecomment-641096443)

Leaving this open for tracking purposes.

**Comment by fsteffi** [comment]
**GitHubTags:**#A11yMAS;#A11yTCS;#Win32;#DesktopApp;#A11y_VSCode;#WCAG1.3.3;#A11ySev2;#VisualStudioCode-Win32-Apr2021;#Benchmark;#Visual Studio Code Client;#Keyboard;

**Comment by HuihuiWu-Microsoft** [comment]
Hi @isidorn , we met the same issue. In our Teams Toolkit extension, we have several buttons on tree items. When user tab to the buttons with keyboard, tooltip will not show:
![image](https://user-images.githubusercontent.com/73154171/195752925-69f095c6-bd4b-48fb-bca3-111eaf1f29cd.png)
Accessibility team has raised a Sev2 bug to us for this issue. Is it possible to fix this?

**Comment by fsteffi** [comment]
same issue seen in bug https://github.com/microsoft/vscode/issues/178410

**Comment by r3m0t** [comment]
It can be done in one line due to a contribution by the Edge team to Chromium https://bugs.chromium.org/p/chromium/issues/detail?id=829352#c17

```diff
diff --git a/src/vs/platform/windows/electron-main/windows.ts b/src/vs/platform/windows/electron-main/windows.ts
index dd832b583fd..503879236c8 100644
--- a/src/vs/platform/windows/electron-main/windows.ts
+++ b/src/vs/platform/windows/electron-main/windows.ts
@@ -132,7 +132,7 @@ export function defaultBrowserWindowOptions(accessor: ServicesAccessor, windowSt  
                        autoplayPolicy: 'user-gesture-required',
                        // Enable experimental css highlight api https://chromestatus.com/feature/5436441440026624
                        // Refs https://github.com/microsoft/vscode/issues/140098
-                       enableBlinkFeatures: 'HighlightAPI',
+                       enableBlinkFeatures: 'HighlightAPI, KeyboardAccessibleTooltip',
                        ...overrides?.webPreferences,
                        sandbox: true
                },
~
```

However, loads of tooltips don't show up, eg this toolbar (top right)-

![image](https://github.com/microsoft/vscode/assets/1485998/9b590d6f-08a8-45d8-9a93-239125cc592e)

Maybe because `:focus` is on an `<a>` element which inherits the `title` attribute from a parent element. This can be reproduced on https://vscode.dev as well.

**Comment by meganrogge** [comment]
@r3m0t thanks for investigating. When I make that change, I don't see it working for any of the buttons. Which work for you?

**Comment by r3m0t** [comment]
Hi, iirc it worked on New Search Editor, Search (text box), Replace (text input) and the icons in the activity bar. By the way, I noticed then that `aria-orientation` is missing in the activity bar.

**Comment by meganrogge** [comment]
@deepak1556 when I test with the above change, it doesn't work for me for any of the buttons. I looked at the docs and it should be supported. Any clue why that would be?

**Comment by r3m0t** [comment]
@meganrogge you need to close and reopen after making the change. Reload Window wouldn't be enough. You can also visit vscode.dev in Edge and it should have the same behaviour.

**Comment by meganrogge** [comment]
Thanks, yeah, I run OSS to test, which should work, but isn't.

**Comment by r3m0t** [comment]
@meganrogge I sent a draft PR and explained there in more detail what works and doesn't work. (Tested on Windows 10)

**Comment by meganrogge** [comment]
Thanks to @r3m0t, this will now work in some cases. @deepak1556 and I have heard that it will work in the web and here's the Chromium bug that should be addressed in a few months: https://bugs.chromium.org/p/chromium/issues/detail?id=1068938

**Comment by meganrogge** [comment]
I am reverting this for now because we don't want multiple tooltips showing up on hover. I'll try to investigate in December.

**Comment by meganrogge** [comment]
Removing these lines fixes the issue, but I worry about making this change so close to the release:

https://github.com/microsoft/vscode/blob/7e5a4aff938d89b21571b63e3dc60a1eff0dd36d/src/vs/base/browser/ui/actionbar/actionViewItems.ts#L226-L232

**Comment by Tyriar** [comment]
@meganrogge the custom hovers are better for accessibility as they're exposed to the screen reader. Generally if the custom hover is being used a `title` should not be present.

**Comment by meganrogge** [comment]
Great, when that has happened, all of these tooltips will be accessible, just as they are for the activity bar / status bar items which rely on custom hovers.

**Comment by Tyriar** [comment]
@meganrogge isn't this separate to adopting it in different places? I would have expected us to need a focus handler to show the hover for that specific case

**Comment by meganrogge** [comment]
Verify that for icon labels with custom hovers, for example, the editor action ones, the tooltip shows up on keyboard focus of the item. 
![Screenshot 2023-11-27 at 2 36 14 PM](https://github.com/microsoft/vscode/assets/29464607/9320d2a9-c0eb-4393-8cea-1776d8cf2e62)

**Comment by meganrogge** [comment]
This can be closed when https://github.com/microsoft/vscode/issues/154364 happens

**Comment by bpasero** [comment]
Ben has adopted the custom hover in our base widgets, so this may now be easier than before?

**Comment by meganrogge** [comment]
Kai mentioned I should consider adding a delay like we have for mouse hovers here. I need to see if this is acceptable by accessibility standards.

**Comment by meganrogge** [comment]
@benibenj it seems like this already works thanks to your changes 

https://github.com/microsoft/vscode/assets/29464607/e0878e90-ab5f-450d-907f-d05183e51a42

**Comment by bpasero** [comment]
When I tested on native Windows explorer, I saw a delay even when using keyboard, so it seems the same delay should apply for keyboard and mouse.

**Comment by isidorn** [comment]
@benibenj awesome work!
